### PR TITLE
chore: update cibuild make formula to build for any OS (windows, linu…

### PR DESCRIPTION
**Pull request by marinsalinas**
…x, macos)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -191,11 +191,7 @@ More Info: https://github.com/golang/dep
...
 
     ```bash
-<<<<<<< HEAD
-    git clone terraform-providers/terraform-provider-nutanix.git $GOPATH/src/github.com/terraform-providers/terraform-provider-nutanix
-=======
     git clone https://github.com/terraform-providers/terraform-provider-nutanix.git $GOPATH/src/github.com/terraform-providers/terraform-provider-nutanix
->>>>>>> chore: fix bad urls in readme.md file
     ```
 
 4. Install [golang/dep](https://github.com/golang/dep):
```
Comment (thetonymaster) [inline README.md:175]: dep is not used anymore, is it?

File: README.md [hunk 2]
```diff
@@ -191,11 +191,7 @@ More Info: https://github.com/golang/dep
 3. Clone the repository:
 
     ```bash
-<<<<<<< HEAD
```
Comment (thetonymaster) [inline README.md:194]: merge leftovers

File: GNUmakefile [hunk 3]
```diff
@@ -60,11 +60,10 @@ test-compile:
 # 	go tool cover -html=c.out
 
 cibuild:
-	go build
-#	env GOOS=darwin GOARCH=amd64 go build
-#	env GOOS=windows GOARCH=amd64 go build
-#	env GOOS=linux GOARCH=amd64 go build
-#	env GOOS=linux GOARCH=ppc64 go build
+	go get github.com/mitchellh/gox
```
Comment (thetonymaster) [inline GNUmakefile]: this should be left to a step on the makefile to install all the dependencies, instead of installing it every time this is run

**Review by thetonymaster — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (thetonymaster) [reply README.md:194]: saw it is deleted, disregard it

**Review by thetonymaster — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (marinsalinas) [reply README.md:175]: Correct, I'll remove this step

**Pull request merged**